src/app/content/utils/seoUtils.ts

```diff
@@ -9,23 +14,125 @@ import {
   splitTitleParts,
 } from './archiveTreeUtils';
 
-const getParentPrefix = (node: LinkedArchiveTreeNode | undefined): string => {
+type Services = {
+  intl: AppServices['intl'];
+  loader: AppServices['archiveLoader'];
+};
+
+const domParser = new DOMParser();
+
+export const getParentPrefix = (node: LinkedArchiveTreeNode | undefined, intl: IntlShape, includeTitle: boolean = false): string => {
   if (!node) {
     return '';
   }
 
   if (archiveTreeSectionIsChapter(node)) {
-    const number = getArchiveTreeSectionNumber(node);
-    return `Ch. ${number} `;
+    const number = getArchiveTreeSectionNumber(node).trim();
+    const name = getArchiveTreeSectionTitle(node);
+    const prefixVariant = includeTitle ? 'with-name' : 'without-name';
+    return intl.formatMessage({id: `i18n:metadata:title:${prefixVariant}`}, {number, name});
```

Comment:
The difference is that we will be able to write a script that will search through the keys and find those that are not used in application
```suggestion
    const i18nTitle = includeTitle ? 'i18n:metadata:title:with-name' : 'i18n:metadata:title:without-name';
    return intl.formatMessage({id: i18nTitle}, {number, name});
```